src/register/RegistrationPage.jsx

```diff
@@ -543,6 +556,44 @@ class RegistrationPage extends React.Component {
       }
     }
 
+    const formFields = Object.keys(this.props.fieldDescriptions).map((fieldName) => {
+      const fieldData = this.props.fieldDescriptions[fieldName];
+      if (fieldData.name === 'country' && fieldData.type === 'select') {
```

Comment:
As mentioned by Waheed we can remove the second check
```suggestion
      if (fieldData.name === 'country') {
```